### DISCUSSION TITLE
Better flares crafted by the Mechanic

### DISCUSSION THREAD

**Discussion by bookslayer10** [FakeFishGames/Barotrauma#13703] [Ideas and suggestions]
The current flares and glowsticks aren't that useful for distracting enemies, but they are also cheap to craft and don't take any skills or talents to do so, which means they will always need to be not very powerful. A more powerful distraction tool would need to be locked behind skills and talents. Therefore, I propose the addition of a new, more powerful and more expensive type of flare.

The upgraded flare would immediately attract all the medium or smaller monster enemies in a large vicinity around it, even if they are already attacking a different target. It will distract an entire hoard at once. It will also be fired out at a large velocity when deployed, causing the distracted enemies to chase after it for a large distance. Once hostiles reach the flare, it wouldn't take long to be destroyed.

Giving this new unlock to Mechanic makes sense, specifically as a talent in the second row of generic talents due to its utility for all playstyles. It would be useful for mechanics in all skill trees, as it would be effective while both exploring caves and repairing submarine leaks.

The flares would be decently expensive for each shot, probably costing some metal, maybe some alien blood as a scent-based distraction, definitely flash powder for the bright light. Firing the flares out of a shotgun for the long range deployment makes sense, but it might not work well due to the shotgun normally being hitscan. A dedicated flare gun would be more work to add but would be better overall.

**Comment by jigardi** [comment]
Have my upvote!  I'm working on a flare gun mod for this exact reason.

I'm not sure I agree with how powerful you want this item to be,  but I agree that it's an are in need of some improvement.
